corehq/apps/users/templates/users/partials/edit_role_modal.html

```diff
@@ -1,7 +1,8 @@
 {% load i18n %}
 {% load hq_shared_tags %}
 
```

Comment:
This commit basically added the Can View / Can Edit columns shown here
![screen shot 2019-02-20 at 1 20 57 pm](https://user-images.githubusercontent.com/716573/53118324-762f3700-3512-11e9-982b-2d2c564a7dd9.png)

And adding the separate Other Settings `<legend>` (the new styling happens in a subsequent PR btw)
![screen shot 2019-02-20 at 1 21 07 pm](https://user-images.githubusercontent.com/716573/53118380-96f78c80-3512-11e9-94a7-2e876191421c.png)